frontend/src/components/auth/Login.tsx

```diff
@@ -30,13 +30,32 @@ const Login = (): React.ReactElement => {
   const [isInvalid, setInvalid] = useState(false);
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const subscriptionErrorMessage = "Your subscription has expired.";
 
   const onLogInClick = async () => {
-    const user: AuthenticatedUser = await authAPIClient.login(email, password);
-    if (!user) {
+    try {
+      const user: AuthenticatedUser = await authAPIClient.login(
+        email,
+        password,
+      );
+      setAuthenticatedUser(user);
+    } catch (error: any) { // eslint-disable @typescript-eslint/no-explicit-any 
       setInvalid(true);
+      let testString = "";
+      for (
+        let index = 0;
```

Comment:
This was pretty stupid, but basically comparing the strings with === was not working. I have tried trimming, replacing white space etc, without it working. I checked every character uni code and it was identical, I would say the issue might be due to types of the axiom?  
![image](https://user-images.githubusercontent.com/37082847/186693621-605ef016-78a6-4be2-b938-1c4b49274db7.png)